docs/topics/wasm/wasm-js-interop.md

```diff
@@ -221,6 +221,46 @@ external class User : JsAny {
 }
 ```
 
+### Array interoperability
+
+You can directly transform JavaScript's `JsArray<T>` into Kotlin's native `Array` or `List` types and vice versa.
```

Comment:
I'd replace "transform" to "copy", and drop unnecessary "directly". Would make it a bit more clear that original JsArray is not affected.